source/_integrations/wallbox.markdown

```diff
@@ -44,6 +44,7 @@ The {% term integration %} adds the following sensors:
 The {% term integration %} adds the following number entities:
 
 - Max charging current
+- Max ICP current
```

Comment:
what is ICP current?

Reply:
Sorry, missed your remarked. The ICP Current (term used by the manufacturer refers to the total power available to the load balancing system of the wallbox). I will add this info to the markdown

Reply:
Ah, I see. Thank you for clarifying.